packages/ui/src/components/PersonalInviteButton.tsx

```diff
@@ -61,22 +62,13 @@ export function PersonalInviteButton() {
   }, [doCopy, inviteLink, userDisplayName]);
 
   return (
-    <Button
-      hero
-      onPress={handleInviteButtonPress}
-      borderRadius="$xl"
-      width="100%"
-      justifyContent="space-between"
-    >
-      <XStack gap="$xl" paddingHorizontal="$m" alignItems="center">
-        <View>
-          <Icon type="Send" size="$l" color="$background" />
-        </View>
-        <Text flex={1} color="$background" fontSize="$l">
-          Invite Friends to Tlon
-        </Text>
-        <Icon type="ChevronRight" size="$l" color="$background" />
-      </XStack>
+    <Button hero onPress={handleInviteButtonPress}>
+      <Button.Icon>
+        <Icon type="Link" />
+      </Button.Icon>
+      <TlonText.Text color="$background" size="$label/l">
```

Comment:
Used TlonText.Text here instead of Button.Text because Button.Text needs to be 100% wide—I didn't want to completely refactor Button to determine if its children contained both text and an icon to conditionally style that variant